### ISSUE TITLE
new maintainer

### ISSUE THREAD

**Issue by auvipy** [niwinz/django-jinja#240]

**Comment by niwinz** [comment]
In the same way of https://github.com/niwinz/django-redis/issues/400

**Comment by auvipy** [comment]
I am interested

**Comment by tiltec** [comment]
Should we try to get this into Jazzband, too?

(For reference: https://github.com/jazzband-roadies/help/issues/176)

**Comment by niwinz** [comment]
First of all sorry for long silence, i was sick among other priorities.
@auvipy if you still interested, confirm please and i give you all permissions. also send me your pypi username for give you permission to release packages.

**Comment by auvipy** [comment]
auvipy and auvipy :)

**Comment by clintonb** [comment]
Has the handoff happened? If so, can we get a new release to PyPI, please?

**Comment by auvipy** [comment]
I will prepare a new release after getting the access

**Comment by niwinz** [comment]
@auvipy you should have all permissions (repo and pypi), ping me if something is missing

**Comment by auvipy** [comment]
i got them all. thanks

**Comment by tomhamiltonstubber** [comment]
Thanks so much @niwinz for maintaining this for so long!

**Comment by wizpig64** [comment]
@niwinz

Hello Andrey,

I'd like to apply to be a co-maintainer for django-jinja, so that I can push version 2.7.0 to pypi. Most of the code since the last was written by me, and merged by @auvipy, but the community is still waiting on a release (#269). I'd like to be the one to do it, given my investment in the project so far. I also wouldn't mind spending some more time closing/resolving some of the open issues here on github, as there are a lot that are fixed now, but have been left open.

I have experience packaging another project of mine on pypi, and I promise to test things out on the 'test' pypi first. Here's my account there: https://pypi.org/user/wizpig64/

Thanks for your time

**Comment by niwinz** [comment]
Hi @wizpig64 

Looks ok for me, Is there any objection @auvipy ?
If this is not replied in one week I will proceed to give maintainer permissions to @wizpig64 

Thanks @wizpig64 for stepping on it.

**Comment by wizpig64** [comment]
@niwinz thanks for getting back to me!

**Comment by auvipy** [comment]
nope no object

**Comment by niwinz** [comment]
@wizpig64 what is your pypi username?

**Comment by wizpig64** [comment]
@niwinz wizpig64!

**Comment by wizpig64** [comment]
The releases 2.7.1 and 2.8.0 now live on PyPI. Thanks again for adding me!

I can't push tags to github, but it's a low priority compared to the real package releases.

```
>$ git tag 2.7.1 e23e0c9; git tag 2.8.0 3ce3bfc; git push niwinz 2.7.1 2.8.0
ERROR: Permission to niwinz/django-jinja.git denied to wizpig64.
```

**Comment by niwinz** [comment]
> The releases 2.7.1 and 2.8.0 now live on PyPI. Thanks again for adding me!
> 
> I can't push tags to github, but it's a low priority compared to the real package releases.
> 
> ```
> >$ git tag 2.7.1 e23e0c9; git tag 2.8.0 3ce3bfc; git push niwinz 2.7.1 2.8.0
> ERROR: Permission to niwinz/django-jinja.git denied to wizpig64.
> ```

Hmm strange, looks like you din't accpeted the invitation, maybe is related to this issue?
![image](https://user-images.githubusercontent.com/843689/120989994-68a02880-c780-11eb-930a-111dc8c8032e.png)

**Comment by wizpig64** [comment]
Oh of course, I don't know how I missed the email notification. Thanks again :)
